broker/etc/artemis-broker.xml

```diff
@@ -133,14 +133,6 @@
          </address-setting>
       </address-settings>
```

Comment:
queues are automatically created using these settings? When a device publishes to *event/BUMLUX* then the broker will automatically create a queue called *event/BUMLUX* and consumers can read events using a receiver link to the dispatch router with a source address of *event/BUMLUX*?

Reply:
Yes. It should for our use case (and in demo mode we don't need more security). Integration tests are passing